src/Components/Blazor/WebAssembly.Authentication/src/Models/RemoteAuthenticationState.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Authentication
+{
+    /// <summary>
+    /// Represents the minimal amount of authentication state to be preserved during authentication operations.
+    /// </summary>
+    public class RemoteAuthenticationState
+    {
+        /// <summary>
+        /// Gets or sets the return url to redirect the application to after a successful authentication operation.
```

Comment:
```suggestion
        /// Gets or sets the URL to which the application should redirect after a successful authentication operation.
```

Also minor nit: please replace `url` with `URL` in the following line.